keyboards/sanctified/dystopia/config.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2021 Quentin Melis
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x514D // "QM"
```

Comment:
```suggestion
#define VENDOR_ID       0x5357 // "SW"
```
I think this would make more sense.

Reply:
That's true, I had put my initials but it makes more sense that way